tools/gn

```diff
@@ -1253,15 +1246,15 @@ def parse_args(args):
 
   parser.add_argument(
       '--enable-impeller-vulkan-playgrounds',
-      default=False,
+      default=True,
```

Comment:
The GN invocations all live in this repo, so if this flag is a no-op now, and is no longer passed in the json files, then it can be safely removed from here.

Reply:
Done